src/main/java/de/adesso/anki/battle/protocol/states/RocketInFront.java

```diff
@@ -1,9 +1,9 @@
-package com.states;
-
-public class RocketInFront extends GameState{
-
-	public RocketInFront(int meters, String type) {
-		// TODO Auto-generated constructor stub
-	}
-
-}
+package de.adesso.anki.battle.protocol.states;
+
+public class RocketInFront extends GameState{
+
+	public RocketInFront(int meters, String type) {
+		// TODO Auto-generated constructor stub
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=22&issues=AWKWIFgW1B6WA4TguGLD&open=AWKWIFgW1B6WA4TguGLD)

[ISSUE_KEY]: AWKWIFgW1B6WA4TguGLD